x/rewards/types/genesis.go

```diff
@@ -31,5 +35,20 @@ func (gs GenesisState) Validate() error {
 			at all times. amount of timers found: %v`, len(timeEntries))
 	}
 
+	for _, sub := range gs.IprpcSubscriptions {
```

Comment:
also check for doubles

Reply:
done. see https://github.com/lavanet/lava/commit/dc29e1a1e0181b1110dac3e10733bc10915a321e